distrobox-init

```diff
@@ -196,6 +196,22 @@ mount_bind() (
 	return 0
 )
 
+# In openSUSE images, zypper is configured to ignore recommended packages
+# (i.e., weak dependencies). This means very few packages are installed,
+# which may sound good, but results if a much worse out-of-the-box experience
+# for users that, for instance, want to run GUI apps in their distroboxes.
+# In fact, we expect that most users would want recommended packages (which
+# are enabled by default in "regular" openSUSE/SUSE installations) in their
+# boxes, so let's re-enable them by default.
+#
+# If wanting to keep them disabled, the user can define to false the
+# DBX_INIT_ZYPPER_NO_RECOMMENDS env variable.
+[ "${DBX_CONTAINER_INIT_ZYPPER_NORECOMMENDS:-}" = "" ] && DBX_CONTAINER_INIT_ZYPPER_NORECOMMENDS=false
+if [ -f /etc/zypp/zypp.conf ]; then
```

Comment:
Was also thinking, instead of this check, we could move the `sed` inside the pkg manager part, so we can standardize when I'll add the other distros unminimization

Reply:
Sounds fine... The only thing i'm not sure is when you say "instead of this check". I can move the check and the sed in the package management "section", but I still need the check, don't I?

Reply:
Yes in the "zypper" section let's call it 😄 
In was referring to the file existence check, I think if you have zypper you mist have zypper.conf right?
Just a thought, we can leave the check as it is no problem 

If was just referring to the file check, obv the variable check remains :+1:

Reply:
Ah, sure! Sorry, it was actually even quite obvious that you were... But for some reason I thought about the variable check. Sorry for the noise! Will do :-)